funcbench/README.md

```diff
@@ -28,8 +28,56 @@ The benchmark can be triggered by creating a comment in a PR which specifies a b
 /funcbench <branch/commit> <benchmark function regex>
 ```
 
+#### Setup
+
 The comment is handled by [comment-monitor](https://github.com/prometheus/test-infra/tree/master/tools/commentMonitor) and then the parsed arguments are handed over to funcbench(if using Github Actions) or to [prombench](https://github.com/prometheus/test-infra/tree/master/prombench) if using funcbench with GKE.
 
+- Create GitHub actions workflow file (see below) that is executed when an `repository_dispatch` event is on.
+- Also add GKE auth info (PROJECT_ID / ZONE / CLUSTER_NAME) as secrets
+- Read LAST_COMMIT_SHA / BRANCH / PR_NUMBER / REGEX from event payload into environment variables
+- Ignore this if your benchmark lasts less than 60 minutes.
+  - Provide a GitHub token as a secret, here named `PERSONAL_TOKEN`, to avoid conflicts with default `GITHUB_TOKEN`.
```

Comment:
I don't see any second mention of `PERSONAL_TOKEN`. Is something missing?

Reply:
No, it's only set in the workflow file.